src/Workload/Microsoft.NET.Sdk.Maui/Microsoft.NET.Sdk.Maui.csproj

```diff
@@ -131,4 +131,73 @@
     <Copy SourceFiles="@(_Files)" DestinationFolder="$(DotNetSdkManifestsDirectory)$(MSBuildProjectName.ToLowerInvariant())" />
   </Target>
 
+  <!-- These are the darc dependency items we need to solicit feeds from  -->
+  <ItemGroup>
+    <_DarcDependency Include="Microsoft.Dotnet.Sdk.Internal" Version="$(MicrosoftDotnetSdkInternalPackageVersion)" />
+    <_DarcDependency Include="Microsoft.NETCore.App.Ref" Version="$(MicrosoftNETCoreAppRefPackageVersion)" />
+    <_DarcDependency Include="Microsoft.Android.Sdk.Windows" Version="$(MicrosoftAndroidSdkWindowsPackageVersion)" />
+    <_DarcDependency Include="Microsoft.MacCatalyst.Sdk" Version="$(MicrosoftMacCatalystSdkPackageVersion)" />
+    <_DarcDependency Include="Microsoft.macOS.Sdk" Version="$(MicrosoftmacOSSdkPackageVersion)" />
+    <_DarcDependency Include="Microsoft.iOS.Sdk" Version="$(MicrosoftiOSSdkPackageVersion)" />
+    <_DarcDependency Include="Microsoft.tvOS.Sdk" Version="$(MicrosofttvOSSdkPackageVersion)" />
+    <_DarcDependency Include="$(MicrosoftNETWorkloadEmscriptenSdkDarcName)" Version="$(MicrosoftNETWorkloadEmscriptenPackageVersion)" />
+    <_DarcDependency Include="Microsoft.AspNetCore.Authorization" Version="$(MicrosoftAspNetCoreAuthorizationPackageVersion)" />
+  </ItemGroup>
+
+  <!-- For each dependency item, execute darc get-asset to list out the version info with feeds -->
+  <Target Name="_DarcGetAssetNugetFeeds" AfterTargets="Build" Outputs="%(_DarcDependency.Identity)">
+
+    <Exec Command="darc get-asset --name %(_DarcDependency.Identity) --version %(_DarcDependency.Version) --max-age 90"
```

Comment:
I think emsdk eventually wants to not change though in that case, they would be on nuget rather than in any feed but it would be older than 90 days...eventually...maybe

Reply:
Good to know - i think as of now this won't expressly fail if there's no feeds in the output (or rather if the command returns standard output saying it can't find any), but I'm not sure if it returns a 0 exit code in those cases or not, so I should probably test that scenario here too.

It sounds like if it can't find recent enough assets that just ignoring it would be ok at least, since as you mention, it should be on nuget.org which will be in the output regardless here.